cmd/ntpcheck/cmd/utils.go

```diff
@@ -83,9 +87,21 @@ func signFile(fileName string) {
 	fmt.Printf("#h %s\n", leaphash.Compute(string(data)))
 }
 
+func stripZeroes(num float64) string {
+	s := fmt.Sprintf("%.2f", num)
+	return strings.TrimRight(strings.TrimRight(s, "0"), ".")
+}
+
+func toByteArray(i uint32) (arr [4]byte) {
+	binary.BigEndian.PutUint32(arr[0:4], i)
+	return
+}
+
 // ntpDate prints data similar to 'ntptime' command output
-func ntpDate(remoteServerAddr string, remoteServerPort string, requests int) error {
+func ntpDate(remoteServerAddr string, remoteServerPort string, requests int, clientTransmitOffset time.Duration, userDate string, blurb bool) error {
```

Comment:
You pass `blurb` bool, but use global `explain` variable in the code instead. No bueno :-)